traffic_ops/testing/api/v4/jobs_test.go

```diff
@@ -36,6 +36,18 @@ func TestJobs(t *testing.T) {
 		GetTestJobs(t)
 		GetTestInvalidationJobs(t)
 		JobCollisionWarningTest(t)
+		GetTestJobsByValidData(t)
+		GetTestJobsByInvalidData(t)
+		CreateTestJobsInvalidDS(t)
+		CreateTestJobsAlreadyExistTTL(t)
+		CreateTestJobswithPastDate(t)
```

Comment:
nit: change `w` in with to capital letter

Reply:
Fixed.